content/en/dora_metrics/setup/deployments.md

```diff
@@ -116,6 +116,18 @@ There are two requirements for calculating Change Lead Time:
 1. Both the Git repository URL and commit SHA are provided when sending deployment events.
 2. Your repository metadata is being [synchronized to Datadog](#synchronize-repository-metadata-to-datadog).
 
+### Breakdown metrics
+
+DataDog also provides the following breakdown metrics, which represent the different stages since the commit is made until it is deployed.
```

Comment:
I think we need to define what a "commit PR" is in this context, as there's not a clear correlation in Git. We can probably say that, if a commit `C` is pushed on a branch `B`, and a PR `P` is opened with `B` as the base branch, the commit `C` is related to the PR `P` (something like this)

Reply:
Maybe the following: A commit is associated with a PR if the commit is first introduced to the target branch when merging that PR.